lib/pbench/server/api/__init__.py

```diff
@@ -17,41 +18,69 @@
 from pbench.common.logger import get_pbench_logger
 from pbench.server.api.resources.query_apis.elasticsearch_api import Elasticsearch
 from pbench.server.api.resources.query_apis.query_controllers import QueryControllers
+from pbench.server.database.database import Database
 from pbench.server.api.resources.query_apis.query_month_indices import QueryMonthIndices
+from pbench.server.api.auth import Auth
+
+from pbench.server.api.resources.pbench_users import (
+    RegisterUser,
+    LoginAPI,
+    UserAPI,
+)
 
 
 def register_endpoints(api, app, config):
     """Register flask endpoints with the corresponding resource classes
     to make the APIs active."""
 
     base_uri = config.rest_uri
-    app.logger.info("Registering service endpoints with base URI {}", base_uri)
+    logger = app.logger
+
+    # Init the the authentication module
+    token_auth = Auth()
+    token_auth.set_logger(logger)
```

Comment:
Although you can call static methods on an object, you're really "supposed" to call them as `Auth.set_logger()`; that's a more clear indication to the reader of what's happening.